activerecord/test/cases/tasks/postgresql_rake_test.rb

```diff
@@ -195,21 +195,56 @@ def setup
         'adapter'  => 'postgresql',
         'database' => 'my-app-db'
       }
+      @filename = "awesome-file.sql"
 
+      ActiveRecord::Base.dump_schemas = :schema_search_path
       ActiveRecord::Base.stubs(:connection).returns(@connection)
+      @connection.expects(:schema_search_path)
```

Comment:
I don't think we should have expectations in the `setup` method. This is something which should be related to a test-case.